### DISCUSSION TITLE
how can I pass jaxprTrace as index of list in jax.scan

### DISCUSSION THREAD

**Discussion by Tanman1234** [jax-ml/jax#6663] [Q&A]
I am working on ILQR algorithm so is there any way to pass jaxPrTrace as a index of list

**Comment by jakevdp** [comment]
No, traced values cannot be used as list indices. Your options, which may or may not be possible depending on the context, are either to change your call such that the traced value is static/concrete, or to convert the list to a `jax.numpy` array.
